lib/evm_database.rb

```diff
@@ -20,6 +20,9 @@ class EvmDatabase
   ORDERED_CLASSES = %w(
     RssFeed
     MiqWidget
+    MiqAction
+    MiqEventDefinition
+    MiqPolicySet
```

Comment:
The policy seeding relays on MiqAction and MiqEventDefinition being seeded before. 
Now that this patch only has MiqPolicySet.seed I can drop this since it is lexicographically last or should I keep it for explicitness @gmcculloug ?